src/Components/UiComponents/SendMessageBar.jsx

```diff
@@ -27,11 +30,17 @@ export default function SendMessageBar() {
     setImage(e.target.files[0]);
   };
 
-  //When user clicks submit, push new message to backend (have to change when we set up room ID)
+  /*
+   * Submitting data will happen in 3 steps
+   * 1) If there is image, first upload to firebase to retrieve URL.
+   * 2) POST message to chatroom_messages to get message ID.
+   * 3) POST image to chat_images with the URL and ID
+   *
+   * */
   const handleSubmit = async (e) => {
     e.preventDefault();
 
-    if (image) {
+    if (image !== "") {
       const storageRefInstance = storageRef(

```

Comment:
i think the best way to use it is by implementing null as initial state and then doing an if image.length > 0 since the images are an array iirc but if this implementation works then all g